navigation/src/main/java/com/graphhopper/navigation/IntersectionValues.java

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.navigation;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class IntersectionValues {
+    int bearing;
+    boolean entry;
+    boolean in;
+    boolean out;
+
+    // create a List of IntersectionValues from PathDetail
```

Comment:
Shouldn't this be a comment starting with `/**`? IMHO a comment should explain and provide additional  context.

It is obvious to me that a method:
`static List<IntersectionValues> createList(Map<String, Object> intersectionMap)`

returns a list of IntersectionValues from PathDetails :). The less obvious part is, why this is done. 

Here this could explain:
- Merge duplicated intersections coordinates
- Filter to the intersections that are actually on this step/instruction/maneuver